### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by baurmatt** [requests/toolbelt#383]
Quite a few commits happened since the last release 1,5 years ago. Any chance to get a shiny fresh release soon?

I'm specifically looking for https://github.com/requests/toolbelt/commit/0e39ddaa77980afe99d35d8088d675a1ed5db0ef to be included.

**Comment by HenkeA** [comment]
> Quite a few commits happened since the last release 1,5 years ago. Any chance to get a shiny fresh release soon?
> 
> I'm specifically looking for [0e39dda](https://github.com/requests/toolbelt/commit/0e39ddaa77980afe99d35d8088d675a1ed5db0ef) to be included.

I am also interested in that one

**Comment by sinback** [comment]
Yes, Akamai employee here. Please make a release, or tell us how to help! We're especially rooting for commit 9f62095.
